src/components/navigation/HaukiNavigation.scss

```diff
@@ -0,0 +1,8 @@
+.navigation-header {
+  nav {
+    a {
+      padding-left: 0 !important;
```

Comment:
Had to apply this dirty hack; HDS nav component has a bug that prevents a more elegant fix. They promised to fix it soon.